ILSpy.ReadyToRun/ReadyToRunOptionPage.xaml.cs

```diff
@@ -20,10 +20,10 @@
 using System.Windows.Controls;
 using System.Xml.Linq;
 using ICSharpCode.ILSpy.Options;
-
+using ILSpy.ReadyToRun;
 namespace ICSharpCode.ILSpy.ReadyToRun
 {
-	[ExportOptionPage(Title = "ReadyToRun", Order = 40)]
+	[ExportOptionPage(Title = nameof(global::ILSpy.ReadyToRun.Properties.Resources.ReadyToRun),   Order = 40)]
```

Comment:
Why using `ILSpy.ReadyToRun` above and FQ name here?

Reply:
Here namespace is  ICSharpCode.ILSpy.ReadyToRun    !
but resource's namespace is ILSpy.ReadyToRun ! They are in conflict.

Reply:
Well, interesting. Why? Could they be unified?